src/main/java/athena/Athena.java

```diff
@@ -1,21 +1,41 @@
 package athena;
 
 import athena.commands.Command;
+import athena.exceptions.AddException;
+
+import java.util.Scanner;
 
 public class Athena {
-    private static Ui ui;
+    private Ui ui;
+    private Parser parser;
+    private Storage storage;
+    private TaskList oldTask;
 
     public Athena() {
         ui = new Ui();
+        parser = new Parser();
+        storage = new Storage("save.csv", ui);
     }
 
     public static void main(String[] args) {
         Athena athena = new Athena();
+
+        athena.runProgram();
+
+        /*Athena athena = new Athena();
         athena.runProgram();
         Storage storage = new Storage("save.csv", ui);
+        TaskList oldTask = new TaskList();
+        oldTask = storage.loadTaskListData();
         TaskList taskList = new TaskList();
         taskList = storage.loadTaskListData();
         Parser parser = new Parser();
+        try {
+            Command command = parser.parse("add n/ t/ d/16-09 D/2 r/Monday i/high a/Refer to lecture notes");
+            command.execute(taskList, athena.ui);
+        } catch (AddException e) {
+            e.getErrorMessage();
+        }
```

Comment:
We should remove these commented code since we are not using it anymore. If we wanna refer back we can find the old version here on GitHub.